components/why-koda/HeroSection.vue

```diff
@@ -0,0 +1,126 @@
+          {{ $t('whyKoda.embraceTheFutureOfGenerativeArt') }}
+        </p>
+
+        <div class="!mt-10 md:mt-16">
+          <ol class="text-xl md:text-2xl !pl-7 list-decimal w-[300px]">
+            <li
+              v-for="section in sections"
+              :key="section.id"
+              class="!mb-3 border rounded-full px-2 md:!px-3">
+              <NeoButton
+                variant="text"
+                tag="a"
+                no-shadow
+                class="text-xl md:text-2xl"
+                :href="`#${section.id}`">
+                {{ section.name }}
+              </NeoButton>
+            </li>
+          </ol>
+        </div>
+      </div>
+    </div>
+
+    <div
+      class="md:absolute top-0 right-0 md:w-[55%] h-full overflow-hidden bg-neutral-3 dark:bg-neutral-11 max-md:border-b md:border-l">
+      <div
+        class="max-md:py-9 flex flex-col max-md:gap-4 md:gap-8 h-full justify-center">
+        <div
+          v-for="(nfts, index) in nftsByStrip"
+          :key="`strip-${index}`"
+          class="flex max-md:gap-4 md:gap-8"
+          :class="{
+            'transform -translate-x-60': index !== 1,
+            'transform translate-x-4 md:translate-x-8': index === 1,
+          }">
+          <div
+            v-for="nft in nfts"
+            :key="nft.id"
+            class="max-md:min-w-[134px] max-md:h-[134px] md:h-[285px] md:min-w-[285px] relative overflow-hidden border rounded-xl shadow-primary">
+            <BasicImage
+              :src="sanitizeIpfsUrl(nft.meta.image)"
+              :alt="nft?.name" />
+          </div>
+        </div>
+      </div>
+    </div>
+  </section>
+</template>
+<script lang="ts" setup>
+import { NeoButton } from '@kodadot1/brick'
+import { getDrops } from '@/services/fxart'
+import latestEvents from '@/queries/subsquid/general/latestEvents.graphql'
+import chunk from 'lodash/chunk'
+import uniqBy from 'lodash/uniqBy'
+
+defineProps<{
+  sections: { name: string; id: string }[]
+}>()
+
+const NFTS_PER_STRIP = 10
+const AMOUNT_OF_STRIPS = 2
+
+const { isMobileOrTablet: isTouch } = useDevice()
+const { isDarkMode } = useTheme()
+
+const { data: collections } = await useAsyncData(
+  () =>
+    getDrops({
+      limit: 12,
+      active: [true],
+      chain: ['ahp'],
+    }),
+  {
+    transform: (data) => data.map((drop) => drop.collection),
+  },
+)
+
+const { data: nftss } = await useAsyncData(
+  async () => {
+    const { data } = await useAsyncQuery<{ events: unknown[] }>({
+      query: latestEvents,
+      clientId: 'ahp',
+      variables: {
+        limit: 40,
+        where: {
+          nft: {
+            collection: {
+              id_in: collections.value || [],
+            },
+          },
+        },
+      },
+    })
+    return data?.value.events || []
+  },
+  {
+    transform: (events) =>
+      uniqBy(
+        events.map((event) => event.nft),
+        'id',
+      ).slice(0, NFTS_PER_STRIP * AMOUNT_OF_STRIPS),
+  },
+)
```

Comment:
Isn't it better t write usable resolver on the squid side?

Reply:
agree

Reply:
on it

Reply:
this?

Reply:
https://github.com/kodadot/stick/pull/272